### PR TITLE
tools: v6 support for testing

### PR DESCRIPTION
**Pull request by MylesBorins**
- bump mock-fs
- add v6 to travis yaml

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "test": "tap test/*.js"
   },
   "dependencies": {
-    "minimist": "0.0.8"
+    "minimist": "0.0.8",
+    "mock-fs": "^3.9.0"
```
Comment (yoshuawuyts) [inline package.json]: I think you meant to bump the dev dependency, not add a new dep

Comment (TheAlphaNerd) [inline package.json]: good eye

File: package.json [hunk 2]
```diff
@@ -16,10 +16,10 @@
     "test": "tap test/*.js"
   },
   "dependencies": {
-    "minimist": "0.0.8"
+    "minimist": "0.0.8",
```
Comment (yoshuawuyts) [inline package.json]: I don't believe trailing commas are valid JSON 😢

Comment (TheAlphaNerd) [inline package.json]: trying to do too much at once… sigh

**Comment by TheAlphaNerd** [comment]
ping @substack

**Comment by TheAlphaNerd** [comment]
ping @substack

**Pull request closed without merging**